extensions/amp-story/0.1/animation-presets-utils.js

```diff
@@ -83,8 +83,8 @@ export function whooshIn(startX, startY, endX, endY) {
  * @param {StoryAnimationDimsDef} dimensions Dimensions of page and target.
  */
 export function pageIsLargerThanTarget(dimensions) {
```

Comment:
With the change to the implementation, should this be renamed?  `pageIsLargerThanOrEqualToTarget(...)`? 
 `targetFitsWithinPage(...)`?